cli/test/test_cmd_commands.py

```diff
@@ -0,0 +1,113 @@
+from click.testing import CliRunner
+import popper.commands.cmd_version as version
+import popper.commands.cmd_run as run
+import popper.utils as pu
+from pathlib import Path
+import unittest
+import os
+from popper.cli import log
+import shutil
+from test.test_common import *
+
+
+class TestCommandVersion(unittest.TestCase):
+
+    def test_version(self):
+        runner = CliRunner()
+        result = runner.invoke(version.cli)
+        assert result.exit_code == 0
+
+
+class TestSkip(unittest.TestCase):
+
+    init_test_repo()
+    os.chdir(os.environ.get('test_repo_path'))
+
+    def create_workflow_file(self, content):
+        f = open(os.environ.get('test_repo_path')+'/a.workflow', 'w+')
+        f.write(content)
+        f.close()
+
+    def test_skip(self):
+
+        self.create_workflow_file("""
```

Comment:
let's create file-based fixtures so that we can test both yaml and hcl with the same test. Please see [this comment](https://github.com/systemslab/popper/pull/772#pullrequestreview-355561864)